influxdb3_server/src/http.rs

```diff
@@ -353,6 +353,15 @@ impl Error {
                     .body(Body::from(mc_err.to_string()))
                     .unwrap(),
             },
+            Self::WriteBuffer(
+                err @ WriteBufferError::CatalogUpdateError(CatalogError::CatalogUpdatedElsewhere {
```

Comment:
@praveen-influx @hiltontj I made the requested change (checking for table existence before attempting to update the schema) but left the changes to `CatalogError::CatalogUpdatedElsewhere` and the handling for that variant here just in case the schema update attempt ever happens in some other code path in the future. Not sure how likely that would be, but it seems like a harmless change to keep.